backend/sql/initial_schema.sql

```diff
@@ -0,0 +1,103 @@
+create table project
+(
+    project_id  bigserial not null primary key,
+    name        text      not null unique,
+    description text      null
+);
+
+create table test_methodology
+(
+    test_methodology_id bigserial     not null primary key,
+    name                text          not null unique,
+    description         text          null,
+    confluence_link     varchar(1000) null
+);
+
+-- Corresponds to an EXCEL file
+create table experiment
+(
+    experiment_id  bigserial                not null primary key,
+    project_id     bigint                   not null,
+    methodology_id bigint                   not null,
```

Comment:
AFAIU most of `bigint`s can't realistically exceed 2147483647, so we can save some space by changing them to `integer`, can't we?

Reply:
we can